api/src/main/java/org/pmiops/workbench/compliancetraining/ComplianceTrainingServiceImpl.java

```diff
@@ -240,14 +240,21 @@ public DbUser syncComplianceTrainingStatusAbsorb(DbUser dbUser, Agent agent) thr
           enrollments.stream().filter(e -> e.courseId.equals(courseId)).findFirst();
 
       if (maybeEnrollment.isEmpty()) {
-        log.severe(
-            String.format(
-                "User `%s` is not enrolled in Absorb course `%s` for access module `%s`. Users are expected to be automatically enrolled in all courses upon visiting Absorb.",
-                dbUser.getUsername(), courseId, accessModuleName));
-        throw new NotFoundException(
-            String.format(
-                "User %s is not enrolled in Absorb course %s", dbUser.getUsername(), courseId));
+        if (accessModuleName == DbAccessModule.DbAccessModuleName.RT_COMPLIANCE_TRAINING) {
```

Comment:
is it the idea that we only prompt users this error when they're trying to access `RT_COMPLIANCE_TRAINING`?

Reply:
Yes